apps/ui/src/Components/CardMenu/CardMenu.tsx

```diff
@@ -31,14 +39,19 @@ const CardMenu: React.FC<CardMenuProps> = ({
   const handleSelectOption = (option: string) => {
     if (option === MANAGER_OPTIONS.EDIT) {
       handleEditClick();
+    } else if (option === MEMBER_OPTIONS.SEE_RIDES) {
+      handleJumpToRides();
```

Comment:
note that enums are just numbers assigned a name, if there's overlap between enums you might encounter unexpected behavior. Also, this can be turned into an Record<string, () => void> to retrieve the correct function